### PR TITLE
feat: support default branch named main

### PR DESCRIPTION
**Pull request by nornagon**
This should support both 'master' and 'main' as default branches.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -152,7 +152,8 @@ const probotHandler = async (robot: Application) => {
   const backportInformationCheck = async (context: Context) => {
     const pr: Octokit.PullsGetResponse = context.payload.pull_request;
 
-    if (pr.base.ref !== 'master') {
+    // TODO(main-migration): Simplify once branch rename is complete.
+    if (pr.base.ref !== 'master' && pr.base.ref !== 'main') {
```
Comment (MarshallOfSound) [inline src/index.ts:156]: There should be a way to do this without hardcoding.

```suggestion
    if (pr.base.ref !== pr.base.repo.default_branch) {
```

Comment (MarshallOfSound) [reply src/index.ts:156]: Other changes like this in GH apps we maintain that non-electron folks use should probably follow a similar approach.

Comment (MarshallOfSound) [reply src/index.ts]: Just to clarify, we need to make this change everywhere that references "master" or "main".  Every lookup, check or comparison you changed to check for either.  Should be changed to check for `default_branch` instead.  No more hardcoded strings 🥳

Comment (nornagon) [reply src/index.ts]: yeah, updated throughout.

File: src/utils.ts [hunk 2]
```diff
@@ -368,7 +368,7 @@ export const backportImpl = async (
   const noEOLSupport = getEnvVar('NO_EOL_SUPPORT', '');
   if (noEOLSupport) {
     const supported = await getSupportedBranches(context);
-    if (!['master', ...supported].includes(targetBranch)) {
+    if (!['master', 'main', ...supported].includes(targetBranch)) {
```
Comment (MarshallOfSound) [inline src/utils.ts:371]: I think this should be dynamic too, `context` _should_ have some kind of repo info attached to it, you might need to look at the type definition to find it though.

Comment (nornagon) [reply src/utils.ts]: done.

**Review by MarshallOfSound — APPROVED** [review]

**Comment by nornagon** [comment]
@MarshallOfSound merge at will

**Pull request merged**